src/packagedcode/conda.py

```diff
@@ -0,0 +1,234 @@
+#
+# Copyright (c) 2019 nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+import io
+import logging
+import sys
+
+import attr
+from six import string_types
+
+from commoncode import filetype
+from commoncode import fileutils
+from commoncode import saneyaml
+
+from packagedcode import models
+from packageurl import PackageURL
+
+
+"""
+Parse Conda manifests, see https://docs.conda.io/en/latest/
+
+"""
+
+TRACE = False
+
+
+def logger_debug(*args):
+    pass
+
+
+logger = logging.getLogger(__name__)
+
+if TRACE:
+    logging.basicConfig(stream=sys.stdout)
+    logger.setLevel(logging.DEBUG)
+
+    def logger_debug(*args):
+        return logger.debug(' '.join(isinstance(a, string_types) and a or repr(a) for a in args))
+
+
+@attr.s()
+class CondaPackage(models.Package):
+    metafiles = ('meta.yaml',  'META.yml',)
+    filetypes = ('.bz', '.bz2', 'bzip2',)
```

Comment:
I am not sure we want to cast so wide a net. ... Let's chat about this

Reply:
@linexb ping?

Reply:
Thanks, I have removed extra test files. :)

Reply:
@pombredanne  Yep, I will remove "filetypes and minetypes" limit  :)